pandas/tests/arrays/test_integer.py

```diff
@@ -339,6 +339,19 @@ def test_error(self, data, all_arithmetic_operators):
         with pytest.raises(NotImplementedError):
             opa(np.arange(len(s)).reshape(-1, len(s)))
 
+    @pytest.mark.parametrize("zero, negative", [(0, False), (0.0, False), (-0.0, True)])
+    def test_divide_by_zero(self, zero, negative):
+        # https://github.com/pandas-dev/pandas/issues/27398
+        a = pd.array([0, 1, -1, None], dtype="Int64")
+        result = a / zero
+        expected = np.array([np.nan, np.inf, -np.inf, np.nan])
+        if negative:
+            values = [np.nan, -np.inf, np.inf, np.nan]
+        else:
+            values = [np.nan, np.inf, -np.inf, np.nan]
+        expected = np.array(values)
```

Comment:
small nitpick but maybe a little more concise to do:
```python
expected = np.array([np.nan, np.inf, -np.inf, np.nan])
if negative:
    expected *= -1
```

Reply:
Given that CI is passing, I think I'm OK with how things are now :)

Reply:
see tests.arithmetic.test_numeric.adjust_negative_zero